src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -139,8 +139,10 @@ public static void NonExistentAction_Throws_ActionNotSupportedException()
     // The client should throw a TimeoutException
     [Fact]
     [OuterLoop]
+    [ActiveIssue(3464)]// Issue is in the corefx repo
     public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutException()
```

Comment:
I wanted to close #273 since there is nothing more for us to do and I didn't like the idea of having an issue hanging around just to track another issue in someone else's repo?